.gitlab/nersc-gitlab-ci.yml

```diff
@@ -0,0 +1,70 @@
+# Kokkos nightly tests at NERSC machine Perlmutter.
+stages:
+  - test
+  - clean_up
+
+default:
+  tags:
+    - perlmutter-login01
+  before_script:
+    - export SOURCE_DIR=$SCRATCH/.nightly/testing/kokkos
+    - if [ ! -d $SOURCE_DIR ]; then
+    - git clone https://github.com/kokkos/kokkos.git ${SOURCE_DIR}
+    - fi
+    - cd ${SOURCE_DIR} && pwd && git pull && git branch
```

Comment:
Does that mean that the workspace is not cleaned up?

Reply:
Is it unique per run?  what does it look like

Reply:
No this was just to output which branch I was running on in case I wanted to look at the output later. I should probably also add something like `git-sha` to get a unique commit number per run.

Reply:
> Does that mean that the workspace is not cleaned up?

Yeah. I should do that. I am only cleaning the builds and not the entire workspace.